shared/elements/gaia_sim_picker/script.js

```diff
@@ -10,6 +10,13 @@
  * Dependencies: gaia-menu web component.
  */
 window.GaiaSimPicker = (function(win) {
+  var l10n = navigator.mozL10n || document.l10n;
```

Comment:
I'm afraid this will prevent the test in `apps/sharedtest/test/unit/elements/gaia_sim_picker/gaia_sim_picker_test.js` from mocking the l10n lib.